extension/resources/walkthrough/command-palette.md

```diff
@@ -1,3 +1,7 @@
+> ℹ️ The extension's features cannot be accessed until DVC is installed and a
+> DVC project is available in the workspace. Please refer to the
+> [setup page](command:dvc.dvc.showDvcSetup) if you have not setup DVC yet.
+
```

Comment:
Should this be above the title?

Reply:
I added it at the top to better show that this admonition is for the walkthrough as a whole vs just this step. But if it looks buggy to you, I'm happy to move it. 

Comparison of the two:

<img width="738" alt="image" src="https://github.com/iterative/vscode-dvc/assets/43496356/11769012-3dd8-4288-a8c9-c2a51a54a788">

<img width="711" alt="image" src="https://github.com/iterative/vscode-dvc/assets/43496356/58bc415e-fd0a-4e56-923f-2f11e269fa89">

Reply:
Looks and feels better after the title.